modules/ROOT/pages/platform/api/authentication.adoc

```diff
@@ -106,13 +106,13 @@ response = requests.request(
     "https://api.neo4j.io/oauth/token",
     headers={"Content-Type": "application/x-www-form-urlencoded"},
     data={"grant_type": "client_credentials"},
-    auth=HTTPBasicAuth(client_id, client_secret),
-    timeout=10
+    auth=HTTPBasicAuth(client_id, client_secret) <1>
 )
 
 print(response.json())
 ----
 
+<1> `client_id` and `client_secret` credentials should be set to the values shown in the Aura Console
```

Comment:
```suggestion
<1> `client_id` and `client_secret` must be set to the values obtained from the Aura Console.
```

Maybe a synonym for "obtain", but we need to be clear (again, as in the first paragraph) that the values are not shown any longer once you have created them.